clip_benchmark/datasets/multilingual_dataset.py

```diff
@@ -0,0 +1,31 @@
+import json, os
+import torch
+
+class Multilingual_Dataset(torch.utils.data.Dataset):
```

Comment:
downloading captions should be automated though, see builder.py with examples

Reply:
I've automated the download process to a large extent.
However, I find that there are 146 images in the multilingual coco dataset that don't seem to exist in the 2014 split... Could it be that the dataset incorporates captions+images from later splits?
[Original Github](https://github.com/adobe-research/Cross-lingual-Test-Dataset-XTD10)

Reply:
I've made it possible to still execute the evaluation without these, but this should obviously be investigated further.